src/main/java/org/zalando/nakadi/domain/EventPublishingStep.java

```diff
@@ -4,6 +4,7 @@
     NONE,
     VALIDATING,
     ENRICHING,
+    VALIDATING_SIZE,
```

Comment:
https://github.com/zalando/nakadi/pull/515/commits/a06139221c7c0cd45bc7c4798b8b1ae7bd083551 and https://github.com/zalando/nakadi/commit/0a791da5ab7ee644797d845445df532d21516361 and https://github.com/zalando/nakadi/commit/df5b675d645fc83bfdba29751d29b8f85e8160e1